winch/codegen/src/isa/x64/masm.rs

```diff
@@ -418,6 +418,60 @@ impl Masm for MacroAssembler {
     fn jmp(&mut self, target: MachLabel) {
         self.asm.jmp(target);
     }
+
+    fn popcnt(&mut self, context: &mut CodeGenContext, size: OperandSize) {
+        let src = context.pop_to_reg(self, None, size);
+        if self.flags.has_popcnt() {
+            self.asm.popcnt(src, size);
+            context.stack.push(Val::reg(src));
+        } else {
+            let tmp = context.any_gpr(self);
```

Comment:
Would you add a comment that the fallback is based on `MacroAssembler::popcnt32` in https://searchfox.org/mozilla-central/source/js/src/jit/x86-shared/MacroAssembler-x86-shared-inl.h?